### ISSUE TITLE
puppet 4 functions don't seem to work

### ISSUE THREAD

**Issue by logicminds** [nwops/puppet-debugger#49]
Not sure when this broke but can't seem to run puppet 4 functions.

```
Evaluation Error: Error while evaluating a Function Call, Function md5 not defined despite being loaded! at /tmp/puppet_debugger_input20170315-208-1dj17f7.pp:1:1

```

**Comment by logicminds** [comment]
This occurs only when starting the debugger with `puppet debugger`.  the `pdb` command seems to be unaffected.

**Comment by logicminds** [comment]
This was caused by a puppet environment creation issue

**Comment by logicminds** [comment]
fixed in 0.5.1
